tests/tensor/test_extra_ops.py

```diff
@@ -647,18 +634,38 @@ def test_grad(self, x_ndim, repeats_ndim, axis):
             repeats_size = (x_test.shape[axis] if axis is not None else x_test.size,)
         repeats = rng.integers(1, 6, size=repeats_size)
         utt.verify_grad(
-            lambda x: Repeat(axis=axis)(x, repeats),
+            lambda x: repeat(x, repeats, axis=axis),
             [x_test],
         )
 
-    def test_broadcastable(self):
-        x = TensorType(config.floatX, shape=(None, 1, None))()
-        r = Repeat(axis=1)(x, 2)
-        assert r.broadcastable == (False, False, False)
-        r = Repeat(axis=1)(x, 1)
-        assert r.broadcastable == (False, True, False)
-        r = Repeat(axis=0)(x, 2)
-        assert r.broadcastable == (False, True, False)
+    def test_static_shape(self):
+        x = TensorType(config.floatX, shape=(None, 1, 3))()
+        symbolic_r = scalar(dtype="int32")
+
+        r = repeat(x, 2, axis=0)
+        assert r.type.shape == (None, 1, 3)
+
+        r = repeat(x, 2, axis=1)
+        assert r.type.shape == (None, 2, 3)
+
+        r = repeat(x, [2], axis=1)
+        assert r.type.shape == (None, 2, 3)
+
+        r = repeat(x, symbolic_r, axis=1)
+        assert r.type.shape == (None, None, 3)
+
+        r = repeat(x, 1, axis=1)
+        assert r.type.shape == (None, 1, 3)
+
+        r = repeat(x, 2, axis=2)
+        assert r.type.shape == (None, 1, 6)
+
+        r = repeat(x, [2, 2, 2], axis=2)
+        assert r.type.shape == (None, 1, 6)
+
+        # This case could be implemented in the future
+        r = repeat(x, [1, 2, 4], axis=2)
+        assert r.type.shape == (None, 1, None)
```

Comment:
The static shape is this case is just the sum of the repeat values?

You want to do something like try to constant fold the sum of repeats? We could just check if its a constant and grab the data out?

Reply:
We could... we can always COULD